core/src/main/resources/reference.conf

```diff
@@ -184,6 +184,7 @@ backend {
         #concurrent-job-limit = 5
 
         run-in-background = true
+        script-epilogue = ""
```

Comment:
I would propose to delete this line and change the `StandardAsyncExecutionActor` to return `sync` instead of nothing by default.